pkg/chartverifier/certifier.go

```diff
@@ -36,17 +36,26 @@ func (e CheckErr) Error() string {
 	return "check error: " + string(e)
 }
 
+type OpenShiftVersionErr struct {
+	msg string
+}
+
```

Comment:
This requires the below hunk to be adapted as well, and the usage further on of `OpenShiftVersionErr{err.Error()}` to `OpenShiftVersionErr(err.Error())`.

Reply:
Done. bd97de524900698c5badb59f30df389f1b827d63